training_courses.rst

```diff
@@ -0,0 +1,31 @@
+Training Courses
+============================================
+
+JFrog has created the JFrog Academy to host a broad range of free online courses
+surrounding Devops. The Conan team has created the "Conan series" on JFrog
+Academy, which includes several levels of courses covering both beginner
+concepts and advanced scenarios.  
+
+The courses are completely free and self-paced.  They feature interactive
+exercises which walk users through the running of commands, exploring and
+editing of important Conan-related files and directories, and quizzes to invoke
+critical thinking after each section.  
+
+For additional information about the Conan training series, see the original
+blog post announcement here:  
+
+- https://blog.conan.io/2020/09/22/New-conan-training-series.html
+
+For the complete list of dedicated Conan courses, see the Conan series page
+here:  
+
+- https://academy.jfrog.com/series/conan
```

Comment:
The academy's link is invalid, is it correct?

Reply:
Not published yet.

Reply:
also, indeed, just found out the link will be invalid.  actual link will be:
`https://academy.jfrog.com/series/jfrog-conan`
Fixing in pr